packages/preview-middleware/src/base/flp.ts

```diff
@@ -30,7 +30,7 @@ export type EnhancedRouter = Router & {
 /**
  * Default theme
  */
-const DEFAULT_THEME = 'sap_horizon';
+const DEFAULT_THEME = 'sap_fiori_3';
```

Comment:
Do we really need to change the default theme. Can we set the theme by using configuration https://github.com/SAP/open-ux-tools/pull/1311?

Reply:
Is there any way to check ui5 version and set this? i did based on the below chart because 'sap_horizon' is not supported for maintenance version
https://sapui5.hana.ondemand.com/sdk/#/topic/38ff8c27b022475a92b591bcf6262551

Reply:
You can get the UI5 version from the `manifest.json`. Additionally one needs to check also the `yaml` configuration of the `ui5-proxy-middleware`.  Hence we need something like https://github.com/SAP/open-ux-tools/blob/e392caa84142d80d69f474309c754f60d80e3698/packages/ui5-proxy-middleware/src/base/utils.ts#L204

Reply:
then i will pass theme internally during flp initilization. Can i expose the function here
https://github.com/SAP/open-ux-tools/blob/e392caa84142d80d69f474309c754f60d80e3698/packages/ui5-proxy-middleware/src/index.ts#L1

Reply:
No, please create a new function in the `preview-middleware`

Reply:
This will be handled in a different PR.